config/webpack.config.prod.js

```diff
@@ -415,7 +415,7 @@ module.exports = {
                         // it's runtime that would otherwise be processed through "file" loader.
                         // Also exclude `html` and `json` extensions so they get processed
                         // by webpacks internal loaders.
-                        exclude: [/\.(js|mjs|jsx|ts|tsx)$/, /\.html$/, /\.json$/],
+                        exclude: [/\.(js|mjs|jsx|ts|tsx|snap)$/, /\.html$/, /\.json$/],
```

Comment:
Should we make the same change in `webpack.config.dev.js`?  Probably not as big of a deal, but just asking.

Reply:
Only prod is used for making the docs site, so I think it should be fine, but we might as well?